src/layouts/PageLayout/PageLayout.stories.tsx

```diff
@@ -0,0 +1,153 @@
+
+export default {
+  component: PageLayout,
+  parameters: {
+    layout: 'fullscreen',
+  },
+  tags: ['autodocs'],
+  argTypes: {
+  },
+  render: args => <PageLayout {...args}/>,
+} satisfies Meta<PageLayoutArgs>;
+
+const title1 = 'Page Title';
+const actions1 = (
+  <>
+    <Button kind="tertiary">Tertiary Button</Button>
+    <Button kind="secondary">Secondary Button</Button>
+    <Button kind="primary">Primary Button</Button>
+  </>
+);
+const CustomInput: React.ComponentProps<typeof Select>['Input'] = props => (
+  <Input {...props} icon="bell" iconLabel="Bell"/>
+);
+const projects = {
+  p1: 'Connection/Project name',
+  p2: 'Connection/Project name 2',
+  p3: 'Connection/Project name 3',
+};
+const selectOptions = (
+  Object.entries(projects).map(([projectKey, projectName]) =>
+    <Select.Option key={projectKey} itemKey={projectKey} label={projectName}/>
+  )
+);
+const title2 = (
+  <Select
+    label="Select project"
+    placeholder="Select project"
+    defaultSelected={projects.p1}
+    options={selectOptions}
+    Input={CustomInput}
+  />
+);
+
+const header1 = (
+  <PageLayout.Header title={title1}>
+    {actions1}
+  </PageLayout.Header>
+);
+
+const header2 = (
+  <PageLayout.Header titleSelect={title2}>
+    {actions1}
+  </PageLayout.Header>
+);
+
+const body = (
+  <PageLayout.Body>
+    <p>Content Area. Notice this has no external padding and no breadcrumbs; those are added by AppLayout.</p>
+  </PageLayout.Body>
+);
+
+type DefaultTabOption = {
+  index: number,
+  className?: string,
+};
+const defaultTabOptions: DefaultTabOption[] = [1,2,3,4].map(index => { 
+  return { index };
+});
+
+type TabsArgs = React.ComponentProps<typeof Tabs>;
+type TabWithTriggerProps = React.PropsWithChildren<Partial<TabsArgs>> & {
+  options?: undefined | Array<DefaultTabOption>,
+  defaultActiveTabKey?: undefined | string,
+};
+const TabWithTrigger = (props: TabWithTriggerProps) => {
+  const { options = defaultTabOptions, defaultActiveTabKey, ...tabContext } = props;
+  
+  const [activeTabKey, setActiveTabKey] = React.useState<undefined | string>(defaultActiveTabKey);
+  
+  return (
+    <Tabs onSwitch={setActiveTabKey} activeKey={activeTabKey} {...tabContext}>
+      {options.map(tab => {
+        return (
+          <Tab
+            key={tab.index}
+            data-label={`tab${tab.index}`}
+            tabKey={`tab${tab.index}`}
+            title={`Tab ${tab.index}`}
+            render={() => <PageLayout.Body>Tab {tab.index} contents</PageLayout.Body>}
+            className={tab.className}
+          />
+        )
+      })}
+    </Tabs>
+  );
+};
+
+// TODO: The defaultActiveTabKey option is not working atm
+// See https://github.com/fortanix/baklava/issues/261
+const tabs1 = (
+  <TabWithTrigger defaultActiveTabKey="1" />
+);
+
+export const PageLayoutStandard: Story = {
+  args: {
+    children: (
+      <>
+        {header1}
+        {body}
+      </>
+    ),
+  },
+};
+
+export const PageLayoutSelect: Story = {
```

Comment:
Instead of "Select", let's use the term "scope", or maybe "scope switcher". It's meant to be a control where you can see/change your current "scope" (like a project, connection, etc.)